engine/source/private/rendering/camera.hpp

```diff
@@ -0,0 +1,28 @@
+﻿#pragma once
+#include "vk_types.hpp"
+
+#include <SDL/SDL_events.h>
+
+namespace lumina
+{
+    class Camera
+    {
+    public:
+        float3 velocity {};
+        float3 position {};
+        float speed {20.0f};
+
+        float pitch {};
+        float yaw {};
```

Comment:
roll?

Reply:
Not needed for current camera implementation